docs/_layouts/default.html

```diff
@@ -1,6 +1,20 @@
 <html>
   <head>
     <title>{{ page.title }}</title>
+    <!-- Google Tag Manager -->
```

Comment:
Info: This change is based off of this guide https://developers.google.com/tag-manager/quickstart